core/actions/email/action.ts

```diff
@@ -28,11 +29,12 @@ export const action = defineAction({
 	params: {
 		schema: z
 			.object({
-				recipient: z
+				recipientEmail: z.string().email().describe("Recipient email address").optional(),
```

Comment:
It's not obvious using the form that the member takes precedence. Is there any way to give this a tooltip explaining that? Or to give this field some styling (like lower opacity or a strikethrough) when the member field has a value?

Reply:
I don't really know how this feature is intended to be used so this may be wrong, but I thought I remembered someone requesting to use an pub value with `CoreSchemaType.Email` for this. So does this also need the little pub value substitution button like the member field has?

Reply:
Woops yeah, thanks for catching!

Reply:
@kalilsn The recipient email address field will now be disabled when the recipient member field has a value, reflecting the precedence rules!

<img width="510" alt="Screenshot 2025-02-04 at 3 20 36 PM" src="https://github.com/user-attachments/assets/363a751c-8ccc-44e5-9aea-345fabb7364f" />